src/main/java/rx/internal/schedulers/CachedThreadScheduler.java

```diff
@@ -24,39 +24,39 @@
 import rx.subscriptions.*;
 
 public final class CachedThreadScheduler extends Scheduler implements SchedulerLifecycle {
```

Comment:
I didn't rename it because it describes the implementation mechanism of the scheduler instead of where it was being used. Since the ThreadFactory was decoupled from this class, it made sense to switch the naming to instead reflect how the scheduler was being used instead of how it was implemented.

I'd prefer to keep the naming of these classes as-is for that reason, unless you feel strongly otherwise.